tests_e2e/test_suites/images.yml

```diff
@@ -101,11 +99,8 @@ images:
    mariner_2_arm64:
       urn: "microsoftcblmariner cbl-mariner cbl-mariner-2-arm64 latest"
       locations:
-         AzureCloud: ["eastus"]
          AzureChinaCloud: []
          AzureUSGovernment: []
-      vm_sizes:
-         - "Standard_D2pls_v5"
```

Comment:
were the size restrictions on arm64 vms removed?

Reply:
it causing issues in canary region, without that it's working everywhere